content/docs/examples/bookinfo/index.md

```diff
@@ -66,42 +66,38 @@ To start the application, follow the instructions corresponding to your Istio ru
 If you use GKE, please ensure your cluster has at least 4 standard GKE nodes. If you use Minikube, please ensure you have at least 4GB RAM.
 {{< /tip >}}
 
-1. Change directory to the root of the Istio installation directory.
-
-1.  Bring up the application containers:
-
-    *   If you are using [manual sidecar injection](/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection),
-        use the following command
-
-        {{< text bash >}}
-        $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo.yaml@)
-        {{< /text >}}
-
-        The `istioctl kube-inject` command is used to manually modify the `bookinfo.yaml`
-        file before creating the deployments as documented [here](/docs/reference/commands/istioctl/#istioctl-kube-inject).
+1.  Change directory to the root of the Istio installation directory.
 
-    *   If you are using a cluster with
-        [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection)
-        enabled, label the `default` namespace with `istio-injection=enabled`
+1.  The default Istio installation uses [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection).
+    Label the namespace that will host the application with `istio-injection=enabled`:
 
-        {{< text bash >}}
-        $ kubectl label namespace default istio-injection=enabled
-        {{< /text >}}
+    {{< text bash >}}
+    $ kubectl label namespace default istio-injection=enabled
+    {{< /text >}}
 
-        Then simply deploy the services using `kubectl`
+1.  Deploy your application using `kubectl` as usual:
```

Comment:
Avoid using "as usual". We don't know what is usual to our readers. Opt instead for more direct explicit language:
```suggestion
1.  To deploy your application, use this `kubectl` command:
```

Reply:
I meant to emphasize the magic the auto-injection does.

Reply:
Removed "as usual"